README.md

```diff
@@ -276,81 +278,383 @@ schedule is listed as N/A.
-  | $`1\text{ in }10`$        | $`\approx  4.77 `$ |
-  | $`1\text{ in }100`$       | $`\approx  9.85 `$ |
-  | $`1\text{ in }1{,}000`$   | $`\approx 15.79 `$ |
-  | $`1\text{ in }10{,}000`$  | $`\approx 22.58 `$ |
-  | $`1\text{ in }100{,}000`$ | $`\approx 30.22 `$ |
-
-- The worst-case in-memory size of the indexes is $`O(n)`$.
-
-  The total in-memory size of all indexes depends on the index type,
-  which is determined by the `confFencePointerIndex` field of
-  `TableConfig`. The in-memory size of the various indexes is described
-  in reference to the size of the database in [*memory
-  pages*](https://en.wikipedia.org/wiki/Page_%28computer_memory%29 "https://en.wikipedia.org/wiki/Page_%28computer_memory%29").
-
-  `OrdinaryIndex`  
-  An ordinary index stores the maximum serialised key for each memory
-  page. The total in-memory size of all indexes is proportional to the
-  average size of one serialised key per memory page.
-
-  `CompactIndex`  
-  A compact index stores the 64 most significant bits of the minimum
-  serialised key for each memory page, as well as 1 bit per memory page
-  to resolve clashes, 1 bit per memory page to mark overflow pages, and
-  a negligible amount of memory for tie breakers. The total in-memory
-  size of all indexes is approximately 66 bits per memory page.
+For a discussion of how the sizes of these components are determined by
+the table configuration, see [Fine-tuning Table
+Configuration](#fine_tuning "#fine_tuning").
 
 The total size of an LSM-tree must not exceed $`2^{41}`$ physical
 entries. Violation of this condition *is* checked and will throw a
 `TableTooLargeError`.
 
-### Implementation
+#### Fine-tuning Table Configuration <span id="fine_tuning" class="anchor"></span>
+
+`confMergePolicy`  
+The *merge policy* balances the performance of lookups against the
+performance of updates. Levelling favours lookups. Tiering favours
+updates. Lazy levelling strikes a middle ground between levelling and
+tiering, and moderately favours updates. This parameter is explicitly
+referenced in the documentation of those operations it affects.
+
+`confSizeRatio`  
+The *size ratio* pushes the effects of the merge policy to the extreme.
+If the size ratio is higher, levelling favours lookups more, and tiering
+and lazy levelling favour updates more. This parameter is referred to as
+$`T`$ in the disk I/O cost of operations.
+
+`confWriteBufferAlloc`  
+The *write buffer capacity* balances the performance of lookups and
+updates against the in-memory size of the table. If the write buffer is
+larger, it takes up more memory, but lookups and updates are more
+efficient. This parameter is referred to as $`B`$ in the disk I/O cost
+of operations. Irrespective of this parameter, the write buffer size
+cannot exceed 4GiB.
+
+`confMergeSchedule`  
+The *merge schedule* balances the performance of lookups and updates
+against the consistency of updates. The merge schedule does not affect
```

Comment:
I'd try and find a different word that consistency here. In a DB context, consistency makes people think of C from ACID, which is not what we mean.

Here we mean something like how smooth, even, not spiky the I/O traffic is. In normal English "consistent" would be fine for this meaning, but not in this context.

Reply:
Good point.